docs/conceptual-guides/distributions.md

```diff
@@ -0,0 +1,89 @@
+---
+myst:
+  html_meta:
+    "description": "A Plone distribution is a pre-packaged version of Plone that includes specific features, themes, modules, and configurations."
+    "property=og:description": "A Plone distribution is a pre-packaged version of Plone that includes specific features, themes, modules, and configurations."
+    "property=og:title": "Plone distributions"
+    "keywords": "Plone 6, distribution, plone.distribution"
+---
+
+(plone-distributions-label)=
+
+# Plone distributions
+
+```{versionadded} Plone 6.1
+```
+
+A Plone {term}`distribution` is a pre-packaged version of Plone that includes specific features, themes, modules, and configurations.
+It is a convenient way to get a specific type of website up and running quickly, as the distribution includes everything needed to run that type of site.
+
+```{seealso}
+To create your own distribution, see {doc}`/developer-guide/create-a-distribution`.
+```
+
+
+## Built-in distributions
+
+Plone comes with two built-in distributions, which correspond to the two Plone user interfaces.
+
+[`plone.volto`](https://github.com/plone/plone.volto)
+:   Create a Plone site with the Volto frontend.
+
+[`plone.classicui`](https://github.com/plone/plone.classicui) 
+:   Create a Plone site with the Classic UI frontend.
+
```

Comment:
@stevepiercy I don't feel super strongly about this, but I had intentionally used the names of the distributions themselves, not the names of the Python packages which are used to distribute them.

See:

- https://github.com/plone/plone.volto/blob/main/src/plone/volto/distributions.zcml#L9
- https://github.com/plone/plone.classicui/blob/main/src/plone/classicui/distributions.zcml#L7

<img width="889" alt="Screenshot 2024-11-03 at 7 43 58 PM" src="https://github.com/user-attachments/assets/b4df424c-9621-47c4-9ab9-75f4d808b438">

Reply:
@stevepiercy I was already thinking `default` is a bad name for the distribution that's configured in plone.volto. "default" is a status more than an identity. And since we have multiple installation methods, there are different defaults in different situations! I'll prepare a PR in plone.volto to change the internal name of the distribution to `volto`.

The launch page is implemented as a standalone js app in https://github.com/plone/plone.distribution/tree/main/frontend ... you might need to get some tips from Érico on how to do dev on it, since I don't see anything about that in the readme for the package.

Reply:
I created the following issues:

- https://github.com/plone/plone.distribution/issues/99
- https://github.com/plone/plone.volto/issues/161
- https://github.com/plone/plone.classicui/issues/18

Resolving this conversation.